### PR TITLE
iiod/init/iiod.service.cmakein: Add support for extra iiod options

### PR DESCRIPTION
**Pull request by mhennerich**
This patch adds support to source iiod options via /etc/default/iiod

Signed-off-by: Michael Hennerich <michael.hennerich@analog.com>

### CODE REVIEW

File: iiod/init/iiod.service.cmakein [hunk 1]
```diff
@@ -1,14 +1,17 @@
+# SPDX-License-Identifier: LGPL-2.1+
+#
 # iiod - Systemd init script
 #
-# Copyright (C) 2016 Analog Devices Inc.
+# Copyright (C) 2016-2021 Analog Devices Inc.
```
Comment (nunojsa) [inline iiod/init/iiod.service.cmakein:5]: The above two options were sneaked in this patch...

**Review by nunojsa — APPROVED** [review]
Just a minor comment from side. I would do the SPDX identifier and the Coypright in a different patch...

**Comment by mhennerich** [comment]
Pushed V2:
Split into two patches

File: iiod/init/iiod.service.cmakein (hunk 1 above)
Comment (mhennerich) [reply iiod/init/iiod.service.cmakein:5]: ACK

**Pull request merged**